lib/msal-core/src/AuthError.ts

```diff
@@ -21,220 +21,220 @@
   * OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
   */
 
- import { Constants } from "./Constants";
- import { ErrorMessage } from "./ErrorMessage";
- import { Utils } from "./Utils";
-  /**
-  * @hidden
-  *
-  * General error class thrown by the MSAL.js library.
-  */
- export class AuthError extends Error {
-     error: string;
-     errorDesc: string;
-
-     constructor(error: string, detailMessage: string) {
-         super(detailMessage);
-         this.error = error;
-         this.errorDesc = detailMessage;
-         this.name = "AuthError";
-         this.stack = new Error().stack;
- 
-         Object.setPrototypeOf(this, AuthError.prototype);
-     }
-      static createUnexpectedError(errDesc: string) {
-         return new AuthError(ErrorMessage.unexpectedError.code, errDesc);
-     }
- }
- 
- /**
-  * @hidden
-  *
-  * Error thrown when there is an error in the client code running on the browser.
-  */
- export class ClientAuthError extends AuthError {
-     constructor(error: string, errorDesc: string) {
-         super(error, errorDesc);
-         this.name = "ClientAuthError";
-
-         Object.setPrototypeOf(this, ClientAuthError.prototype);
-     }
- 
-     static createEndpointResolutionError(errDesc: string) : ClientAuthError {
-         var errorMessage = "Error: could not resolve endpoints. Please check network and try again.";
-         if (!Utils.isEmpty(errDesc)) {
-             errorMessage += " Details: " + errDesc;
-         }
-         return new ClientAuthError(ErrorMessage.endpointResolutionError.code, errorMessage);
-     }
- 
-     static createMultipleMatchingTokensInCacheError(scope: string) : ClientAuthError {
+import { Constants } from "./Constants";
+import { ErrorMessage } from "./ErrorMessage";
+import { Utils } from "./Utils";
+/**
+* @hidden
+*
+* General error class thrown by the MSAL.js library.
+*/
+export class AuthError extends Error {
+    error: string;
+    errorDesc: string;
+
+    constructor(error: string, detailMessage: string) {
+        super(detailMessage);
+        this.error = error;
+        this.errorDesc = detailMessage;
+        this.name = "AuthError";
+        this.stack = new Error().stack;
+
+        Object.setPrototypeOf(this, AuthError.prototype);
+    }
+    static createUnexpectedError(errDesc: string) {
+        return new AuthError(ErrorMessage.unexpectedError.code, errDesc);
+    }
+}
+
+/**
+ * @hidden
+ *
+ * Error thrown when there is an error in the client code running on the browser.
+ */
+export class ClientAuthError extends AuthError {
+    constructor(error: string, errorDesc: string) {
+        super(error, errorDesc);
+        this.name = "ClientAuthError";
+
+        Object.setPrototypeOf(this, ClientAuthError.prototype);
+    }
+
+    static createEndpointResolutionError(errDesc: string): ClientAuthError {
+        var errorMessage = "Error: could not resolve endpoints. Please check network and try again.";
```

Comment:
why is this not in the config file for errors

Reply:
seems like some are in config some are not

Reply:
This should probably be added to the ErrorMessage.ts file.